docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,477 @@
+
+> [!NOTE] 
+> Continuous Patching is in Private Preview as of October 2024.
+
+
+## Use Cases
+
+Here are a few scenarios to use Continuous Patching:
+
+- **Enforcing container security and hygiene:** Continuous Patching enables users to quickly fix OS container CVEs without the need to fully rebuild from upstream.
+- **Speed of Use:** Continuous Patching removes the dependency on upstream updates for specific images by updating packages automatically. Vulnerabilities can appear every day, while popular image publishers may only offer a new release once a month. With Continuous Patching, you can ensure that container images within your registry are patched as soon as the newest set of OS vulnerabilities are detected.
+
+## Preview Limitations
+
+Continuous Patching is currently in private preview. The following limitations apply:
+- Windows-based container images aren’t supported.
+- Only "OS-level" vulnerabilities that originate from system packages will be patched. This includes system packages in the container image managed by an OS package manager such as “apt” and “yum”. Vulnerabilities that originate from application packages, such as packages used by programming languages like Go, Python, and NodeJS are unable to be patched.  
+- End of Service Life (EOSL) images are not supported by Continuous Patching. EOSL images refer to images where the software provider is no longer offering updates, security patches, and technical support. Examples include Debian 8 and Fedora 28. EOSL images will be skipped from the patch despite having vulnerabilities - the recommended approach is to upgrade your image to a supported version.
+
+
+## Prerequisites        
+
+- You can use the Azure Cloud Shell or a local installation of the Azure CLI with a minimum version of 2.15.0 or later. 
+- You have an existing Resource Group with an Azure Container Registry.
+- You have an Azure Container Registry with Tasks enabled (Tasks is not supported in the free tier of ACR) 
+
+## Key Concepts
+Because Continuous Patching in ACR creates a new image per patch, ACR relies on a tag convention to version and identify patched images. The two main approaches are incremental and floating.
+
+### Incremental Tagging
+How It Works
+
+Each new patch increments a numerical suffix (e.g., -1, -2, etc.) on the original tag. For instance, if the base image is python:3.11, the first patch creates python:3.11-1, and a second patch on that same base tag creates python:3.11-2.
```

Comment:
Because tagging conventions are important to distinguish, let's highlight them using inline code blocks to increase visibility and drive home this concept.

Each new patch increments a numerical suffix (e.g., `-1`, `-2`, etc.) on the original tag. For instance, if the base image is `python:3.11`, the first patch creates `python:3.11-1`, and a second patch on that same base tag creates `python:3.11-2`.